src/main/resources/lang/en_US.json

```
@@ -449,6 +449,11 @@
         "disabled": "Magma boss timer messages will no longer be logged.",
         "enabled": "Magma boss timer messages will now be logged."
       },
+      "version": {
+        "help": "Prints the mod's version",
```

Print can be quite vague, specially in translations
```suggestion
        "help": "Shows the mod's version",
```